lua/copilot_cmp/completion_functions.lua

```diff
@@ -55,15 +55,19 @@ methods.getCompletionsCycling = function (self, params, callback)
   local request = self.client.rpc.request
 
   local respond_callback = function(err, response)
-    if err then return err end
-    if not response or vim.tbl_isempty(response.completions) then return end
+    if err then 
+      callback(format_completions({}, params.context, self.formatters))
+      return err 
+    end
+    if not response or vim.tbl_isempty(response.completions) then 
+      callback(format_completions({}, params.context, self.formatters))
```

Comment:
`cmp` seems to continue the flow when the callback is executed, it means `cmp.completion()` got the data response and populated it in the popup list.